.github/workflows/build.yml

```diff
@@ -0,0 +1,93 @@
+name: Pipeline
+on:
+  pull_request:
+    branches:
+      - master
+      - v*
+      - litmus-v2
```

Comment:
You need to provide tags here ?
`types: [edited, opened, synchronize, reopened]`